### PR TITLE
Cart Management

### PR DESCRIPTION
**Pull request by prajapati-priyanka**
-Add cart-page.
-On clicking Add to cart button item goes to cart page and button text changed to Go to cart.
-On Increasing item in the cart page, quantity increases as well as item in the cart also increases same with decreasing quantity.
-Item can be removed from cart page by clicking the delete button. 
-Item can be moved to wishlist page by clicking on Save to wishllist button. If item already exist in the wishlist then on clicking on Save to wishlist button, that item will be only deleted from cart page.
-pricing of every items is also calculated.

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
remove empty braces in some files. good to go

File: src/components/product-card/ProductCard.css [hunk 1]
```diff
@@ -1,6 +1,9 @@
 .card {
   margin: 0;
 }
+.ecommerce-card {
+  width: 19rem;
+}
 .card-with-badge .card-badge {
   position: absolute;
   left: -1px;
```
Comment (palash23jain) [inline src/components/product-card/ProductCard.css:9]: you can avoid using px unit as you are using rem consistently

File: src/index.css [hunk 2]
```diff
@@ -12,10 +12,9 @@
   --secondary-color: #ff723f;
   --light-secondary-color: #ff723f30;
   --text-color: #282c3f;
-  --ratingLessThanThree: rgb(212, 210, 77)
+  --ratingLessThanThree: rgb(212, 210, 77);
```
Comment (palash23jain) [inline src/index.css:15]: you can try keeping color format same throughout.. either hexadecimal or rgb (for consistency)

**Review by palash23jain — APPROVED** [review]
Code looks clean and good to merge.

**Pull request merged**